### PR TITLE
covrpage post

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/posts/2018-12-01-covrpage.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+* A README for the tests/ folder, which is the original report location. `covrpage::covrpage()` sets it up. Target audience: users or collaborators browsing the GitHub repo of your package, possibly guided there by a badge in the main README.
+
+* A vignette, that'll get inserted into the `pkgdown` website of your package, and the [CRAN page](https://cran.r-project.org/web/packages/texPreview/index.html) if/when your package is released on CRAN. `covrpage::use_covrpage_vignette()` sets it up. Target audience: users reading the rendered documentation.
+
+In both cases, you can also ensure the report stays up-to-date by having it deployed from Travis every time you push to your repository.
+
+# How to learn more about `covrpage`?
+
+To read more about getting started with `covrpage` in your own package in a few lines of code only, we recommend checking out the GET STARTED vignette. It explains more how to setup the Travis deploy, mentions which functions power the `covrpage` report and gives more motivation for using `covrpage`.
```
Comment (yonicd) [inline content/posts/2018-12-01-covrpage.md]: link to get started vignette: https://yonicd.github.io/covrpage/articles/get-started.html

File: content/posts/2018-12-01-covrpage.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+And to learn how the information provided by `covrpage` should be read, read the HOW TO READ THE REPORT vignette.
+
+# What did `covrpage` gain from our help?
+
+As [announced recently](https://itsalocke.com/blog/package-support-offer/), we at Locke Data will help a package a month get more widely adopted. In the case of `covrpage`, our main input was:
+
+* the creation of a nice logo by Oz Locke. DESCRIBE LOGO.
```
Comment (yonicd) [inline content/posts/2018-12-01-covrpage.md]: The logo highlights that covrpage extends the covr and testthat packages by helping the user inspect their results.

**Comment by maelle** [comment]
@yonicd are you ok with this version?

**Comment by yonicd** [comment]
looks good!

File: content/posts/2018-12-05-covrpage.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+# How to publish `covrpage` report?
+
+There are two places where you can keep `covrpage` report, and it's advised to use both since they will get seen by different readers:
+
+* A README for the tests/ folder, which is the original report location. `covrpage::covrpage()` sets it up. Target audience: users or collaborators browsing the GitHub repo of your package, possibly guided there by a badge in the main README.
+
+* A vignette, that'll get inserted into the `pkgdown` website of your package, and the [CRAN page](https://cran.r-project.org/web/packages/texPreview/index.html) if/when your package is released on CRAN. `covrpage::use_covrpage_vignette()` sets it up. Target audience: users reading the rendered documentation.
+
+In both cases, you can also ensure the report stays up-to-date by having it deployed from Travis every time you push to your repository.
+
```
Comment (yonicd) [inline content/posts/2018-12-05-covrpage.md]: # Interrogate

See a new package on GitHub that you would like to use, but need to feel more comfortable before _commiting_ to it? You can `covrpage_snapshot()` to create a report in a sterile environment, without affecting your `.libPaths`, and make a more informed decision.

**Pull request merged**